SPECS/vim/vim.spec

```diff
@@ -67,20 +67,22 @@ au BufEnter,BufNew *.py set tabstop=4 shiftwidth=4 expandtab
 if ! isdirectory("~/.vim/swap/")
         call system('install -d -m 700 ~/.vim/swap')
 endif
-set directory=~/.vim/swap//
+set directory=~/.vim/swap/
 " End %{_sysconfdir}/vimrc
 EOF
 
 %check
 sed -i '/source test_recover.vim/d' src/testdir/test_alot.vim
-make test
+sed -i '916d' src/testdir/test_search.vim
```

Comment:
Can we please add a comment here on what we are doing here :)

Reply:
Added comment to changelog.